lib/bitbucket_rest_api/repos/commits.rb

```diff
@@ -28,9 +28,15 @@ def list(user_name, repo_name, branchortag=nil, params={})
       normalize! params
       filter! VALID_KEY_PARAM_NAMES, params
 
-      path = "/2.0/repositories/#{user}/#{repo.downcase}/commits"
+      path = if ButBucket.options[:bitbucker_server]
+               "/1.0/projects/#{user_name}/repos/#{repos}/commits"
+             else
+               "/2.0/repositories/#{user}/#{repo.downcase}/commits"
+             end
+
```

Comment:
Do you know if we will be able to pull a commit back by sha or by branch? That's the current bitbucket cloud functionality.